Cargo.toml

```diff
@@ -53,6 +53,7 @@ url = "1.1"
 
 [target.'cfg(unix)'.dependencies]
 openssl = "0.9"
+core-foundation = { version = "0.4.4", features = ["mac_os_10_7_support"] }
```

Comment:
What's the syntax for this? I've tried `[target.'cfg(macos)'.dependencies]`, but it didn't get picked up.